tutorials/in-app/flingGame.json

```diff
@@ -4725,6 +4758,7 @@
       "nextStepTrigger": {
         "valueEquals": "="
       },
+      "skippable": true,
```

Comment:
I may have forgotten this "skippable" for the "valueEquals" "=" in all tutorials. Can you check other ones to add it too?